readthedocs/restapi/views/model_views.py

```diff
@@ -92,11 +92,16 @@ class ProjectViewSet(UserSelectViewSet):
     model = Project
     pagination_class = api_utils.ProjectPagination
 
+    def get_project_for_user_or_404(self, user, lookup_value):
+        lookup_query = {self.lookup_field: lookup_value}
+        return get_object_or_404(
+            Project.objects.api(user), **lookup_query)
```

Comment:
the queryset should be something like this
```python
queryset = self.get_queryset()
return get_object_or_404(queryset, **lookup_query)
```

Reply:
yeah that would make sense but it's not that i am adding new code, i'm just moving it.

Reply:
As you are changing this, You should make this also!